submissions/adekemiAwomolo/database/init.sql

```diff
@@ -0,0 +1,144 @@
+CREATE DATABASE e_commerce;
+SHOW DATABASES;
+USE e_commerce;
+
+/*categories table*/
+CREATE TABLE categories(
+id INT NOT NULL AUTO_INCREMENT,
+category VARCHAR(200) NOT NULL,
+PRIMARY KEY(id)
+);
+
+INSERT INTO categories(category)
+VALUES('Fresh Fruits'),
+('Vegetables'),
+('Tomatoes'),
+('lettuce');
+
+SELECT * FROM categories;
+
+/*create admin table*/
+
+CREATE TABLE admins(
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(100) NOT NULL,
+email VARCHAR(100) NOT NULL,
+address VARCHAR(255) NOT NULL,
+password VARCHAR(50) NOT NULL,
+phone_number VARCHAR(200) NOT NULL,
+date_created DATETIME,
+PRIMARY KEY(id)
+);
+
+
+INSERT INTO admins(name, email, address, password, phone_number, date_created)
+VALUES('Aneke Paul', 'neke@hotmail.com','america' ,'234typo', '0706099884', now()),
+('Paul Panda', 'pai23@hotmail.com','canada', 'paulpp', '0706099894', now()),
+('Kmei Sanders', 'pai23@hotmail.com','canada', 'lekmpp', '0706033384', now());
+
+SELECT * FROM admins;
+
+/*product_details database*/
+CREATE TABLE products_details(
+id INT NOT NULL AUTO_INCREMENT,
+productCategory_id INT NOT NULL,
```

Comment:
There is category here

Reply:
Good afternoon @adaugochi . I do not understand ma. am I to put in a category table here or to remove it ma